<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_staff_form</class>
 <widget class="QWidget" name="add_staff_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>566</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>692</width>
        <height>529</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QFrame" name="frame_19">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_36">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Staff Picture</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_28">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Staff id</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="upload_s_id_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="placeholderText">
               <string>S001</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_29">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="upload_s_name_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="placeholderText">
               <string>Staff Member Name</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_30">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Gander</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_31">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Department</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_32">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QTextEdit" name="upload_s_address_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTextEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="placeholderText">
               <string>Staff Member's Address</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_33">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Tel. Number</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="upload_s_tel_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="placeholderText">
               <string>07X XX XX XXX</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_34">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>E-mail</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="upload_s_email_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="placeholderText">
               <string>emample : xyz@gmail.com</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_35">
              <property name="styleSheet">
               <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
              </property>
              <property name="text">
               <string>Salary</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="upload_s_salary_2">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <property name="text">
               <string>Rs.45000</string>
              </property>
              <property name="placeholderText">
               <string>Rs. 45000.00</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QFrame" name="frame_25">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="upload_staff_member_pp_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>50</horstretch>
                   <verstretch>50</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(168, 168, 168);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_15">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>13</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="upload_staff_pp">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">/* Default button style */
QPushButton {
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
                 </property>
                 <property name="text">
                  <string>Upload Image</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_16">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>17</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="upload_s_gender">
              <property name="styleSheet">
               <string notr="true">QComboBox{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <item>
               <property name="text">
                <string>Male</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Female</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="upload_s_department">
              <property name="styleSheet">
               <string notr="true">QComboBox{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
              </property>
              <item>
               <property name="text">
                <string>Maintenance</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Kitchen</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Housekeeping</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Banquets</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_20">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_22">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_23">
             <item>
              <widget class="QLabel" name="label_37">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
               </property>
               <property name="text">
                <string>Grama Niladhari Certificate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="s_upload_g_certificate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Default button style */
QPushButton {
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
               </property>
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="view_s_upload_g_certificate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>140</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>140</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(168, 168, 168);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_23">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_22">
             <item>
              <widget class="QLabel" name="label_38">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: #a8a8a8;</string>
               </property>
               <property name="text">
                <string>Police Report</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="s_upload_p_certificate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Default button style */
QPushButton {
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
               </property>
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="view_s_upload_p_certificate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>140</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>140</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(168, 168, 168);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_24">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>4</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="view_s_update_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Default button style */
QPushButton {
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>2</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="save_insert_data">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Default button style */
QPushButton {
    background-color: #4C4C6C;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 



}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
               </property>
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>1</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
